.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "eslintIntegration": true,
```

Comment:
AFAIK, this is an option of VS Code extension, `prettier` CLI is reporting an error when it encounters it in the rc file (IIRC). Can we remove it please?

See https://github.com/strongloop/loopback-next/blob/3698d76825b27175e2533ef378db1ebb5e6952ca/.prettierrc#L1-L6